configure

```diff
@@ -21,7 +21,9 @@ use strict;
 use warnings;
 
 ## Version of OCI's TAO to download
-my $tao_version = '2.2a';
+my $oci_tao_version = '2.2a';
+## Version of DOC Group ACE/TAO to download, uses the ACE version number
+my $doc_tao_version = '6.4.3';
```

Comment:
Is this file going to need to change after each DOC release?  Would it be better to have a copy or symlink on the Vanderbilt server that doesn't have the version number?